docs/src/config/stepper-quickstart.adoc

```diff
@@ -140,15 +141,15 @@ latexmath:[
 ////////////////////////////////////////////
 
 .Units mm
-====
+----
 ............................................
     Stepper         = 200 steps per revolution
     Drive           =   8 micro steps per step
     Motor Teeth     =  30
     Leadscrew Teeth =  90
     Leadscrew Pitch =   5.00 mm per turn
 ............................................
-====
+----
 
```

Comment:
Sometimes it feels like ping pong.
For this example: Steffen misshaped the formatting of this block, then I corrected it, and then you are coming again with this :see_no_evil: :smile:

Reply:
Worst is that I don't understand why I reverted it :facepalm:

FIXED

OT: commenting straight from Codium... Men I like that IDE, not perfect for sure, but so many cool things :)

Reply:
>Worst is that I don't understand why I reverted it 

Because it is not obvious that I changed it. And it was still existing in the other format in `_es`. And my change was more intended as a comment to propagate it to ES as well, but I know it is hard to keep the overview on that mass of files and changes ...

You don't have to fix anything here, I do that. It's just for information/questioning.